.github/workflows/web-service.yml

```diff
@@ -30,96 +45,105 @@ jobs:
-          mkdir -p web-service/assets
-
-          cp ${GITHUB_WORKSPACE}/target/aarch64-unknown-linux-gnu/release/personalization ./web-service/server.bin
-          cp ${GITHUB_WORKSPACE}/web-api/Dockerfile ./web-service/Dockerfile
-          cp ${GITHUB_WORKSPACE}/assets/smbert_v0003/config.toml ./web-service/assets/config.toml
-          cp ${GITHUB_WORKSPACE}/assets/smbert_v0003/vocab.txt ./web-service/assets/vocab.txt
-          cp ${GITHUB_WORKSPACE}/assets/smbert_v0003/model.onnx ./web-service/assets/model.onnx
-          tar -cvf web-service.tar ./web-service
-
-      - name: Upload archive
-        uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v3.1.2
+      - name: Create backoffice artifact
+        uses: ./.github/actions/release_artifact
         with:
-          name: web-service
-          retention-days: 1
-          if-no-files-found: error
-          path: ${{ runner.temp }}/web-service.tar
+          platform: ${{ inputs.platform }}
+          bin_name: "ingestion"
+          archive_name: ${{ env.backoffice_archive }}
+          model_full_name: ${{ inputs.model_name }}_${{ inputs.model_version }}
+
+      - name: Create frontoffice artifact
+        uses: ./.github/actions/release_artifact
+        with:
+          platform: ${{ inputs.platform }}
+          bin_name: "personalization"
+          archive_name: ${{ env.frontoffice_archive }}
+          model_full_name: ${{ inputs.model_name }}_${{ inputs.model_version }}
 
   docker-build:
     runs-on: ubuntu-22.04
     timeout-minutes: 10
-    needs: rust-build
+    needs: services-build
     steps:
       - name: Checkout repository
         uses: actions/checkout@24cb9080177205b6e8c946b17badbe402adc938f # v3.4.0
 
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@e81a89b1732b9c48d79cd809d8d81d79c4647a18 # v2.1.0
-        with:
-          platforms: 'arm64'
-
       - name: Setup Docker Buildx
         uses: docker/setup-buildx-action@4b4e9c3e2d4531116a6f8ba8e71fc6e2cb6e6c8c # v2.5.0
 
-      - name: Download artifacts
-        id: artifacts
-        uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a # v3.0.2
+      - name: Set up QEMU
+        if: inputs.platform == 'arm64'
+        uses: docker/setup-qemu-action@e81a89b1732b9c48d79cd809d8d81d79c4647a18 # v2.1.0
         with:
-          name: web-service
+          platforms: ${{ inputs.platform }}
 
-      - name: Create docker image
+      - name: Images name
         run: |
-          tar -xvf web-service.tar
-          cd ./web-service
           TAG=${{ github.event.inputs.tag }}
           if [ -z "$TAG" ]; then
-            TAG="branch-$(git rev-parse --abbrev-ref HEAD)"
-            if [[ "${TAG}" == "branch-HEAD" ]]; then
-              TAG="$(git rev-parse --short HEAD)"
-            fi
+            TAG="$(git rev-parse --abbrev-ref HEAD)"
+            TAG="$TAG-$(date +"%y%W%u%H%M%S")"
+            TAG="$TAG-$(git rev-parse --short HEAD)"
```

Comment:
❓ even though both git parses use different options, it will usually just result in the same string, like `unify_release_build-23171163035-unify_release_build`. do we actually need both or shouldn't the first be enough?